ozaria/site/views/play/level/PlayLevelView.js

```diff
@@ -884,17 +884,9 @@ class PlayLevelView extends RootView {
       this.surface.showLevel()
     }
     Backbone.Mediator.publish('level:set-time', { time: 0 })
-    if (
```

Comment:
I'm a little confused regarding why this could be deleted. But I also love seeing code get deleted!

Reply:
Yep this solves the issue with dev=true. Basically, the code in the if-block was no longer relevant because this was to skip the legacy intro screen when in dev mode, but now we are keeping the behavior consistent. Also `loadingView.startUnveiling` has been removed since that code is moved to the on click handler of `start` button on the intro modal, i.e. publish of this event: https://github.com/codecombat/codecombat/pull/5505/files#diff-5a00df3717cc9ade14619a5f033f1c9bR900.